DuckDuckGoVPN/DuckDuckGoVPNAppDelegate.swift

```diff
@@ -220,10 +220,13 @@ final class DuckDuckGoVPNAppDelegate: NSObject, NSApplicationDelegate {
         return controller
     }()
 
+    private let internalUserDecider = DefaultInternalUserDecider(store: UserDefaults.appConfiguration)
```

Comment:
Uses the app configuration user defaults to know if the user is internal.  This benefits from KVO so it should stay updated with the main app.